ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -72,6 +72,8 @@
   private final List<NetworkReceive> completedReceives;
   private final List<String> disconnected;
   private final List<String> connected;
+  private final List<SSLTransmission> pendingSslHandshakes;
+  private final Map<String, Long> sslHandshakeTimer;
```

Comment:
Why not avoid the map completely? Just use a list of objects of a private class:

```java
private class PendingHandshakeTransmission {
  SSLTransmission transmission;
  Long pendingSinceMs;
}
```